cdap-data-fabric/src/main/java/co/cask/cdap/data2/datafabric/dataset/DatasetServiceClient.java

```diff
@@ -147,6 +148,17 @@ public DatasetMeta getInstance(String instanceName) throws DatasetManagementExce
     return GSON.fromJson(response.getResponseBodyAsString(), SUMMARY_LIST_TYPE);
   }
 
+  Collection<DatasetSpecificationSummary> getInstances(Map<String, String> properties)
```

Comment:
Add a javadoc for what the `properties` is for.

Reply:
added javadoc.